PaymentKit/PTKView.h

```diff
@@ -42,5 +42,6 @@
 @property (nonatomic, strong) UIColor *customFontColor;
 @property (nonatomic, weak) id <PTKViewDelegate> delegate;
 @property (nonatomic, readwrite) PTKCard *card;
+@property (nonatomic, assign) BOOL shouldResignFirstResponderOnValidInput;
```

Comment:
Can you just AppleDoc this property? Other than that, I'm good to go.

I'm goign to file an issue to document all of these properties.